### PR TITLE
Release 5.1

### PR DESCRIPTION
**Pull request by AlenPelin**

### CODE REVIEW

**Comment by AlenPelin** [comment]
@alexshyba for some reason I cannot assign you as a reviewer on this like I did with others

File: uniform.config.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ const dotenv = require('dotenv');
 const defaults = {
   UNIFORM_API_SITENAME: 'uniform-jss-kit',
   UNIFORM_PUBLISH_TARGET: 'fake',
+  UNIFORM_SSR_ENGINE: 'nextjs',
+  UNIFORM_API_KEY: 'eefe326b-aff1-4154-9ae8-2beb85d4b8cb',
```
Comment (alexshyba) [inline uniform.config.js:11]: @AlenPelin ?

File: cloudflare-worker/wrangler-webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+//const path = require('path')
```
Comment (alexshyba) [inline cloudflare-worker/wrangler-webpack.config.js:1]: @AlenPelin dead code

**Review by alexshyba — CHANGES_REQUESTED** [review]

File: cloudflare-worker/wrangler-webpack.config.js (hunk 2 above)
Comment (AlenPelin) [reply cloudflare-worker/wrangler-webpack.config.js:1]: not a release blocker

**Pull request closed without merging**